src/components/common/ShareStructure/ShareStructures.vue

```diff
@@ -8,8 +8,6 @@
 
     <ShareStructureShared
       :isEditMode="isEditMode"
-      :isCorrection="isCorrectionFiling"
-      :incorporationApplication="getCorrectedFiling"
```

Comment:
If the "isCorrection" flag isn't set then this shared component uses "originalShareStructure" (line 14) instead of "incorporationApplication" (line 12). This is exactly what we want: a single way to provide the original share structure (from the entity snapshot object).

Reply:
Also, I will import the latest version of this shared component, which is cleaned up to not use these 2 obsolete fields.